codesign/bin/codesign.dart

```diff
@@ -80,6 +81,11 @@ Future<void> main(List<String> args) async {
           'e.g. supply `--gcs-upload-path=gs://flutter_infra_release/ios-usb-dependencies/ios-deploy/<commit>/ios-deploy.zip`'
           ' if you would like to codesign ios-deploy.zip, which has a google cloud bucket path of flutter_infra_release/ios-usb-dependencies/ios-deploy/<commit>/ios-deploy.zip to be uploaded to',
     )
+    ..addOption(
+      kCredentialsPathOption,
+      help: 'The file path of the password file in file system. The password file stores sensitive passwords.\n'
+          'sensitive passwords include <CODESIGN-APPSTORE-ID>, <CODESIGN-TEAM-ID>, and <APP-SPECIFIC-PASSWORD> \n',
```

Comment:
Secrets management usually keeps one single value per file.

Reply:
Thanks for the info!